lib/src/marker_cluster_layer.dart

```diff
@@ -28,18 +28,18 @@ class _MarkerClusterLayerState extends State<MarkerClusterLayer>
     with TickerProviderStateMixin {
   Map<int, DistanceGrid<MarkerClusterNode>> _gridClusters = {};
   Map<int, DistanceGrid<MarkerNode>> _gridUnclustered = {};
-  MarkerClusterNode _topClusterLevel;
-  int _maxZoom;
-  int _minZoom;
-  int _currentZoom;
-  int _previousZoom;
-  double _previousZoomDouble;
-  AnimationController _zoomController;
-  AnimationController _fitBoundController;
-  AnimationController _centerMarkerController;
-  AnimationController _spiderfyController;
-  MarkerClusterNode _spiderfyCluster;
-  PolygonLayer _polygon;
+  late MarkerClusterNode _topClusterLevel;
+  int? _maxZoom;
+  late int _minZoom;
+  int? _currentZoom;
+  int? _previousZoom;
+  double? _previousZoomDouble;
+  AnimationController? _zoomController;
+  AnimationController? _fitBoundController;
+  AnimationController? _centerMarkerController;
+  AnimationController? _spiderfyController;
+  MarkerClusterNode? _spiderfyCluster;
+  PolygonLayer? _polygon;
```

Comment:
It think it's worth revisiting all of the nullable types here to check if they can be `late` instead. If they are set to a non-nullable type in initState and are never changed to null afterwards it should be fine to make them late.

Reply:
Lmk which ones specifically. PolygonLayer for example does not seem to be a good candidate as it is set to null

Reply:
Here's a PR for your PR 😄 .

I think it's the easiest way to point out which fields can be nullable:

https://github.com/bsneider/flutter_map_marker_cluster/pull/2